<?xml version="1.0"?>
<launch>
    <arg name="robot_name" default="car1"/>
    <arg name="robot_peer_name" default="car2"/>
    <arg name="role" default="leader"/>
    <!--Double_Assembled-->
    <!--Single_Assembled-->
    <!--Single_AMR-->
    <arg name="init_state" default="Single_AMR"/>

    <node pkg="solamr" type="fsm.py" name="fsm" output="screen">
        <param name="roslaunch_path_single_amr" value="$(find solamr)/launch/run_single_AMR_$(arg robot_name).launch"/>
        <param name="roslaunch_path_double_amr" value="$(find solamr)/launch/run_double_AMR_$(arg robot_name).launch"/>
        <param name="robot_name" value="$(arg robot_name)"/>
        <param name="robot_peer_name" value="$(arg robot_peer_name)"/>
        <param name="role" value="$(arg role)"/>
        <param name="init_state" value="$(arg init_state)"/>
        <param name="frequency" value="10"/>
        <param name="dummy_test" value="false"/>
    </node>
</launch>